mono/mini/il2cpp-c-types.h

```diff
@@ -8,7 +8,10 @@
 
 #define IL2CPP_MONO_PUBLIC_KEY_TOKEN_LENGTH	17
 
-typedef struct Il2CppType Il2CppMonoType;
+//Converted to typedef
+#define MonoType Il2CppType
```

Comment:
Why not a typedef?

Reply:
typedef redefines the type so we must be including the mono headers. Maybe I should stop including those and change to a typedef?